Source/WebCore/dom/Document.cpp

```diff
@@ -1327,6 +1329,22 @@ MediaQueryMatcher& Document::mediaQueryMatcher()
     return *m_mediaQueryMatcher;
 }
 
+const RenderStyle* Document::mediaQueryDefaultStyle() const
+{
+    if (m_cachedMediaQueryDefaultStyle == nullptr) {
```

Comment:
WebKit coding style:
```suggestion
    if (!m_cachedMediaQueryDefaultStyle)
```